src/NuGet.Core/NuGet.Commands/RestoreCommand/CompatibilityIssue.cs

```diff
@@ -168,6 +271,27 @@ private string FormatMessage(string message, string supports, string noSupports)
                     sb.Append($" {FormatFramework(AvailableFrameworks.Single())}");
                 }
             }
+            else if(AvailableFrameworkRuntimePairs.Any())
+            {
+                sb.AppendFormat(supports);
+
+                if (AvailableFrameworkRuntimePairs.Count > 1)
+                {
+                    // Write multiple frameworks on new lines
+                    foreach (var framework in AvailableFrameworkRuntimePairs.Select(e => FormatFramework(e.Framework, e.RuntimeIdentifier))
+                        .OrderBy(s => s, StringComparer.CurrentCultureIgnoreCase))
+                    {
+                        sb.Append(Environment.NewLine);
+                        sb.Append($"  - {framework}");
```

Comment:
nit: there might be a helper around for building these multi-line message strings